config/undergraduate_thesis_or_project.yml

```diff
@@ -182,10 +182,7 @@ migration_nodes:
       name:
         field:
           name: degree_name
-        method:
-          - Mapping::Degree.remapped_field_split
-          - degree_name
-          - degree_field
+        method: Mapping::Degree.unprocessed
```

Comment:
@vantuyls and I discussed this yesterday.. and came to the conclusion that degree should be processed here, but the values should be empty of the split was unsuccessful (instead of throwing an error).. and this concept applies to all of the work_types that use the `remapped_field_split` method.